examples/ui/ui_texture_slice.rs

```diff
@@ -0,0 +1,87 @@
+//! This example illustrates how to create a button that has its image sliced
```

Comment:
This could use more meat. Why might a user want to do this?

Reply:
I updated the docs, is that enough ?

Reply:
Yep this is fine.